.gitignore

```diff
@@ -19,3 +19,4 @@ massif.*
 
 # tilt
 tilt_modules/
+.idea
```

Comment:
To remove the files:
` git rm --cached -r .idea`

And to prevent from pushing them:
` echo .idea >> .git/info/exclude

Reply:
Note we don't want to modify `.gitignore` here that is why I suggested the local exclusions instead.